packages/transmute-did/docs/revocation.md

```diff
@@ -0,0 +1,42 @@
+## Revocation of a DID
```

Comment:
Can we use hyperlinks or symlinks instead of duplicating documentation?

Reply:
actually, we can just eliminate the docs folder in the did package.